spec/models/ability_spec.rb

```diff
@@ -35,6 +35,11 @@
 
     # Test abilities for not signed in users
     context 'when user is not signed in' do
+      let(:open_registration_period) { create(:registration_period, start_date: Date.current - 6.days) }
+      let(:open_registration_conf) { create(:conference, registration_period: open_registration_period) }
```

Comment:
I would suggest a variable name that better indicates what it is about. Eg
`let(:conference_with_open_registration)`

Same for the following.